app/experimenter/experiments/constants/nimbus.py

```diff
@@ -178,12 +178,26 @@ class NimbusTargetingConfig:
 TARGETING_MOBILE_NEW_USER = NimbusTargetingConfig(
     name="New Users on Mobile",
     slug="mobile_new_users",
-    description=("New users on mobile"),
-    targeting=("is_already_enrolled || days_since_install < 7 || days_since_update < 7"),
+    description=("New users on mobile, i.e users who installed the app within a week"),
+    targeting=("is_already_enrolled || days_since_install < 7"),
     desktop_telemetry="",
     application_choice_names=(Application.FENIX.name, Application.IOS.name),
 )
 
+TARGETING_MOBILE_RECENTLY_UPDATED_USER = NimbusTargetingConfig(
+    name="Recently Updated Users",
+    slug="mobile_recently_updated_users",
+    description=(
+        "Users who updated their app within the last week. This excludes users who are new users"
```

Comment:
Actually you'll probably have to brake this up by hand since `black` won't do that automatically.  Just change it to 

```py
    description=(
        "Users who updated their app within the last week. "
        "This excludes users who are new users"
    )
```

Reply:
Awesome thank you! yeah `code_format` didn't fix this one 😄